### PR TITLE
No shape noise

### PR DESCRIPTION
**Pull request by jpratmarti**
Adding an option in the Covariance stage to set sigma_e to zero when running without shape noise, to get a consistent gaussian covariance.

### CODE REVIEW

File: examples/config/cosmodc2_config.yml [hunk 1]
```diff
@@ -136,7 +136,9 @@ TXTwoPointFourier:
     bandwidth: 200
     apodization_size: 0.0
     cache_dir: ./cache
-
+    
+TXRealGaussianCovariance:
```
Comment (joezuntz) [inline examples/config/cosmodc2_config.yml:140]: We probably don't want to set this for the default pipeline config - you could make a second config file if you like?

Comment (jpratmarti) [reply examples/config/cosmodc2_config.yml:140]: Sure makes sense. Will do now. I will just set it to False and whoever wants to run without shape noise can set it to True.

Comment (jpratmarti) [reply examples/config/cosmodc2_config.yml:140]: Done.

**Comment by joezuntz** [comment]
Thanks - just one question - does everything work out okay when you set sigma_e = 0 - there are no errors or anything when you invert the matrix?  Just wanted to check, then will merge.

**Comment by jpratmarti** [comment]
Yes, I was able to compare the no shape noise Gaussian covariance with the JK one and it's not perfect but not crazy either. Thanks!

**Pull request merged**